sv.c

```diff
@@ -12132,6 +12133,8 @@ Perl_sv_vcatpvfn_flags(pTHX_ SV *const sv, const char *const pat, const STRLEN p
 
         char c;                       /* the actual format ('d', s' etc) */
 
+        bool escape_it   = FALSE;     /* if this is a string should we quote and escape it? */
+
```

Comment:
Extraneous blank line

Reply:
that one is intentional, the list is divided into groups of different declarations. with whitespace separators.